### DISCUSSION TITLE
Sandbox Options 수정은 가능한가요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#663] [5. Q&A]
onTick이 SandboxOptions의 tickInterval 의해 정해진다고 하는데, 해당 부분의 수정이 가능할까요?

![image](https://user-images.githubusercontent.com/5895396/165227333-e42adac4-6748-4aed-8c32-c55b1cdaf029.png)
![image](https://user-images.githubusercontent.com/5895396/165227362-8a9f9806-a7ec-46cc-86dd-e905a2c9a138.png)

위에서는 readonly로 나와 있어서, 혹시 수정하는 방법이 있는 지 여쭤보게 되었습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. 현재  tickInterval 수정 방법이 없어서 deltaTime 값을 이용해서 원하는 시간마다 콜하는 방식으로 사용하면 좋을 것 같습니다.
